monero-outreach-docs/translations/pt-br/PSA/PSA_pt-br_headers.md

```diff
@@ -0,0 +1,36 @@
+# Headers
+
+- Wallet
+  * My Monero is tight - Meu Monero está seguro
+  * And out of sight - E fora de vista
+  * Closed - Fechado
+
+- Exchange
+  * Monero Hero says: - O herói do Monero diz:
+  * Don’t be a Jive Turkey - Não seja cabeça oca
+
+- Learning
+  * there are 52 learning weeks in a year - existem 52 semanas de aprendizado no ano
+
+- Don't Talk
+  * Shut it To The max - Cale-se ao máximo
+
+- Teamwork
+  * Monero Teamwork - Monero é trabalho em equipe
+  * -from start to finish! - -do começo ao fim!
+
+- Don´t share
+  * How about we go back to my place and you give me your xmr private keys? - Que tal se voltarmos para minha casa e você me der suas chaves XMR privadas?
+  * Keep on dreaming, lover boy… - "Continue sonhando, bonitão…

```

Comment:
"Continue sonhando" sounds good, but I think it would be better "Vai sonhando", it is more common expression. If you agree, then change the next line as well

Reply:
Changed. This is a kind of girls expression, not sure if you have noticed the artwork, take a look https://www.monerooutreach.org/monero_best_practices/never-share-your-private-keys.php. I think you know better than guys do :stuck_out_tongue_winking_eye:.